isis/src/base/apps/ckwriter/CkKernelWriter.cpp

```diff
@@ -37,7 +37,7 @@
 #include "IString.h"
 #include "NaifStatus.h"
 #include "Pvl.h"
-#include "SpiceSegment.h"
+#include "CkSpiceSegment.h"
```

Comment:
The SpiceSegment class was separated into two different classes. Might quickly look into merging the two if possible.